### PR TITLE
Hello world in nodejs

### PR DESCRIPTION
**Pull request by szydan**

### CODE REVIEW

File: nodejs/hello-world.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Hello world 
+
+First program in any new language is called a hello world program.
```
Comment (emmaxdan) [inline nodejs/hello-world.md]: The first program in any new language is called a hello world program.

File: nodejs/hello-world.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Hello world program prints "Hello World" text on the screen.
```
Comment (emmaxdan) [inline nodejs/hello-world.md:4]: make sure to put 'the' infront of hello world

File: nodejs/hello-world.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Below an example in nodejs
```
Comment (emmaxdan) [inline nodejs/hello-world.md]: Below is an example in nodejs

File: nodejs/hello-world.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+console.log("Hello World")
+```
+
+To run it we have to first create a file called hello-world.js (the name of the file does not matter) and add the above line and save it.
+Then to execute it we will use the nodejs environment from our shell.
+
+Assuming 
+
+* you have nodejs installed. If not see how to do it [here](../setup/nodejs.md)
+* you have a file called hellow-world.js
```
Comment (emmaxdan) [inline nodejs/hello-world.md]: you have a file called hello-world.js

File: nodejs/hello-world.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* you have a file called hellow-world.js 
+
+Type `node hello-world.js` in your terminal and hit enter
+You shoud see a "Hello world" printed on your screen.
+
+![](img/hello-world.png)
+
+### What has happend 
+
+In ou first program we
```
Comment (emmaxdan) [inline nodejs/hello-world.md]: In our first program we

File: nodejs/hello-world.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+You shoud see a "Hello world" printed on your screen.
+
+![](img/hello-world.png)
+
+### What has happend 
+
+In ou first program we 
+
+* used a built-in global object called `console`
+* this object have few methods and we used one called `log`
```
Comment (emmaxdan) [inline nodejs/hello-world.md]: this object has a few methods and we use one called `log`

**Review by emmaxdan — CHANGES_REQUESTED** [review]

**Comment by szydan** [comment]
@emmaxdan corrected typos, next thing I would like to do it to run it together

**Pull request merged**